src/web3-utils.js

```diff
@@ -18,10 +18,5 @@ export function getWeb3(provider) {
   return web3
 }
 
-export function weiToEther(wei) {
-  return wei / 10e17
-}
-
-export function etherToWei(ether) {
-  return ether * 10e17
-}
+// Re-export some utilities from web3-utils
+export { fromWei, isAddress, toChecksumAddress, toWei } from 'web3-utils'
```

Comment:
I don’t like much the naming of these web3.js functions (omitting the “what”), but it’s probably a good idea to not rename for the sake of renaming.

Reply:
I guess the "what" is omitted due to it being an optional setting.